rtdata/languages/default

```diff
@@ -4331,6 +4383,11 @@ TP_WAVELET_OPACITYWL_TOOLTIP;Modify the final local contrast at the end of the w
 TP_WAVELET_PASTEL;Pastel chroma
 TP_WAVELET_PROC;Process
 TP_WAVELET_PROTAB;Protection
+TP_WAVELET_PYRWAVTRC;Contrast profile
+TP_WAVELET_PYRWAVTRC_TOOLTIP;The 'Contrast profile' slider lets you select a series of presets that will significantly modify the effect of the “Variable contrast” curve. The presets range from least to most aggressive in ascending order.\n\nFor the upper preset levels, the visibility of the result in the preview window will depend on the image size and the system configuration. Increase the preview window size if this is the case.
+TP_WAVELET_PYRWAVTRC_CURVE_TOOLTIP;The Variable contrast curve takes into account the information provided by the presets. The mid-point of the x-axis corresponds to the predominant contrast value, irrespective of the detail level and the selected preset. The maximum point, to the right of the x-axis, corresponds to the maximum contrast value. It is recommended to reduce the variable contrast beyond the mid-point.
```

Comment:
This says the variable contrast is affected by the contrast profile, but doesn't explain how. From what I read in the comments, the effect of the variable contrast is reduced with higher presets. The tooltip also doesn't say why the variable contrast should be reduced after the midpoint. I assume it's to avoid harsh contrast and halos. The following suggestion is based on my assumptions. Please adapt it so that the information is correct.

```suggestion
TP_WAVELET_PYRWAVTRC_CURVE_TOOLTIP;The left end of the horizontal axis corresponds to zero contrast, while the right end corresponds to the strongest contrast in the image. The midpoint of the horizontal axis corresponds to the predominant contrast strength across all detail levels.\n\nThe effect of this curve is reduced for higher Contrast profile presets.\n\nTo avoid harsh contrast and halos, refrain from increasing the right half of the curve.
```

Reply:
Here again I don't know what to answer, except that the value 0 is not possible, otherwise it would not be in the decomposition.
I come to believe (it's a joke) that faced with the difficulty of explaining complex things simply, it is better to say nothing